fbchat/utils.py

```diff
@@ -225,3 +231,35 @@ def get_emojisize_from_tags(tags):
         except (KeyError, IndexError):
             log.exception('Could not determine emoji size from {} - {}'.format(tags, tmp))
     return None
+
+def encode_params(data):
```

Comment:
I'm really not a fan of including something like this. I realize that it solves the problem, but as a first, it doesn't work in Python 3, and second, it adds a lot more complexity. As I said, just using `'true'` would solve the problem too. If this doesn't do anything else, I'd suggest removing it

Reply:
I agree it would be simpler. I just didn't like having to remember to put 'true' instead of True xD

Curiously there are some other requests which have a Boolean parameter and those work even without this change.

Reply:
BTW the code of the encode_params comes straight from the source of the requests module

Reply:
Hmm, I think I'll leave it out for now, again because of the added complexity, but if I or somebody else finds another Facebook request where passing `'true'` is *required*, I'll add it back!

Reply:
Also, I found that there were other issues, eg. when sending images, and a few unicode related bugs in python 2